src/mds/agg_mds/datastore/__init__.py

```diff
@@ -54,11 +82,3 @@ async def get_commons():
 
 async def get_all_metadata(*args):
     return await client.get_all_metadata(*args)
-
```

Comment:
These are not used. Search will most likely use guppy instead of supporting it directly.

Reply:
Hey Craig, which function(s) are you referring to? It's not clear from the code snippet.

Reply:
Odd I removed the search functions and that was what I was referring to, but not sure why this comment did not get outdated.